app/Http/Controllers/FrontendStatusController.php

```diff
@@ -154,7 +154,6 @@ public function getStatus($statusId) {
 
         return view('status', [
             'status' => $statusResponse,
-            'currentUser' => Auth::user(),
```

Comment:
Ich meine, wir hatten mal gesagt, dass wir `Auth::user()` einmal evaluieren und dann nur mehrmals daraufzurückgreifen, um die Anzahl an DB-Queries kleinzuhalten.

Reply:
I'll agree with you, but in the blades I use auth()-user() for the same reason. If you call this multiple times you get the same model back, so there ia no problem.

Reply:
Okay fair